apps/about/src/about/urls.py

```diff
@@ -15,9 +15,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from django.conf.urls import url
+import sys
+
 from about import views as about_views
 
+if sys.version_info[0] < 3:
+  from django.conf.urls import url
+else:
+  from django.urls import re_path as url
```

Comment:
Could we be "forward" looking? (i.e. making the Django 3 first as this is the future going forward and using the same name)
```
if sys.version_info[0] > 2:
  from django.urls import re_path
else:
  from django.conf.urls import url as ...
```

Reply:
Done.
And re_path() is not backward compatible (compatible till Django 2).

Reply:
It does not answer the question, reading the docs did "The django.conf.urls.url() function from previous versions is now available as django.urls.re_path()."

Reply:
yes, that's why I put that link in the description (please see in How was this patch tested?) @romainr
some lines from the link ->
The django.conf.urls.url() function from previous versions is now available as django.urls.re_path(). The old location remains for backwards compatibility, without an imminent deprecation. The old django.conf.urls.include() function is now importable from django.urls so you can use from django.urls import include, path, re_path in your URLconfs.